serial.c

```diff
@@ -84,21 +96,22 @@ static int check(enum sp_return result) {
 
   switch (result) {
   case SP_ERR_ARG:
-    SDL_LogError(SDL_LOG_CATEGORY_SYSTEM, "Error: Invalid argument.\n");
-    abort();
+    SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "Error: Invalid argument.\n");
```

Comment:
I wasn't seeing any of these messages. According to SDL2/SDL_log.h, the default values log INFO and higher messages for APPLICATION category, but only CRITICAL or higher for (almost) everything else. So logging at SYSTEM level meant that these messages weren't being displayed by default, so I changed them.